vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -88,9 +88,11 @@ import (
 )
 
 const (
-	ApiVersion              = "v1.22"
+	ApiVersion              = "v1.24"
```

Comment:
Which Docker Daemon (1.12/1.13?) supports this API version. Please add a comment.